website/docs/r/function_app_slot.html.markdown

```diff
@@ -248,6 +248,14 @@ A `microsoft` block supports the following:
 
 ---
 
+A `twitter` block supports the following:
+
+* `consumer_key` - (Required) The OAuth 1.0a consumer key of the Twitter application used for sign-in.
+
+* `consumer_secret` - (Optional) The OAuth 1.0a consumer secret of the Twitter application used for sign-in. Cannot be specified with `consumer_secret_setting_name`.
```

Comment:
The `consumer_secret_setting_name` is only available in the new Beta resources, so we don't need to make reference to it here.
```suggestion
* `consumer_secret` - (Optional) The OAuth 1.0a consumer secret of the Twitter application used for sign-in. 
```

Reply:
Done.